crates/kubelet/src/resources/device_plugin_manager/manager.rs

```diff
@@ -0,0 +1,844 @@
+//! The Kubelet device plugin manager. Consists of a `DeviceRegistry` that hosts a registration service for device plugins, a `DeviceManager` that maintains a device plugin client for each registered device plugin, a `NodePatcher` that patches the Node status with the extended resources advertised by device plugins, and a `PodDevices` that maintains a list of Pods that are actively using allocated resources.
```

Comment:
Nit: Line wrap this comment. We generally wrap it at 100 chars, but even just breaking it up a little bit will make this easier to review/read depending on the editor

Reply:
Thanks wrapped all the inline docs: e1f5b841ef